### DISCUSSION TITLE
Error: Unknown version of DB found, version=5

### DISCUSSION THREAD

**Discussion by ghost** [ACINQ/eclair#1953] [Q&A]
I keep getting this error when launching eclair.

![8Nu09i7DiIg-Vaf1A65VnpgT1WeeZad_YduZ8zLDy9E](https://user-images.githubusercontent.com/31393558/133352700-c063741d-0d65-4fc3-b5f3-4be9556a446c.jpg): Unknown version of DB found, version=5

I'm using eclair-node-gui-0.3.3-12ac145.jar

**Comment by t-bast** [comment]
The version you're running is completely outdated.
Plus we removed support for the GUI you're using more than a year ago.
Many things can go wrong in that setup...

**Reply by ghost** [reply]
Do you know if the GUI will someday come back? It's too hard to run it without a GUI

**Reply by t-bast** [reply]
It won't come back. It never was a tool to administer a node with real funds, if you did that you were putting your money at risk. You can use https://github.com/Ride-The-Lightning/RTL if you want a UI for your node.
